src/Engine/Scene/Scene.cpp

```diff
@@ -3,11 +3,12 @@
 #include <algorithm>
 
 Scene::Scene() :
-    m_gameLogicSystem(),
-    m_renderSystem(),
-    m_spatialSystem(),
-    m_collisionSystem(),
-    m_pathfindingSystem(),
+	m_gameLogicSystem(),
```

Comment:
Are these...tabs??

Reply:
I didn't realize that I hadn't set tabbing to spaces after I reinstalled Visual Studio. Made the change in all code I could find.